types/chai-json-pattern/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for chai-json-pattern 1.1
+// Project: https://github.com/damian-szulc/chai-json-pattern, http://chaijs.com
+// Definitions by: Christoph Herb <https://github.com/chrishrb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+// <reference types="chai" />
+
+declare global {
+    namespace Chai {
+        interface Assertion {
+            matchPattern(expected: any): void;
+        }
+    }
+}
+
+declare namespace ChaiJsonPattern {
+    interface Plugin {
+        [key: string]: (target: any) => void;
+    }
+
+    interface ChaiJsonPattern extends Chai.ChaiPlugin {
+        extend(plugin: Plugin): void;
+    }
+}
+
+declare const chaiJsonPattern: ChaiJsonPattern.ChaiJsonPattern;
+export = chaiJsonPattern;
```

Comment:
Use export default to export things from module since that's what package does

Reply:
should be resolved now, thanks 👍